ocaml/sdk-gen/csharp/autogen/src/Session.cs

```diff
@@ -254,36 +226,31 @@ public override string SaveChanges(Session session, string _serverOpaqueRef, Ses
             throw new Exception("The method or operation is not implemented.");
         }
 
-        public Proxy XmlRpcProxy { get; private set; }
-
         public JsonRpcClient JsonRpcClient { get; private set; }
 
+        [Obsolete("Use opaque_ref instead.")]

```

Comment:
These Obsolete properties were removed before the SDK generation code came back to this repo, how have they been re-introduced in this code? I think these changes may have been done off the wrong base? I'm reviewing commit by commit and right now I'm a the third commit, but please have a look and rebase as necessary before I proceed further.

Reply:
Indeed this was an issue.

I went through every file change in this PR, and I can confirm only this file seems to have the issue.

I have double checked it and reverted the changes that were not meant to go in. (commit coming soon)